tests/e2e/set-ffi-env-e2e

```diff
@@ -261,13 +261,15 @@ fi
 echo
 info_message "Checking if QM already installed"
 info_message "=============================="
+QM_INST="$(rpm -qa qm)"
 QM_STATUS="$(systemctl is-enabled qm 2>&1)"
```

Comment:
When `qm` is not installed in the environment,  error will occur when running to this line

Reply:
Would you like to add only an error message here or also to exit the tool if the QM is not installed
?

Reply:
When qm is not installed, `set-ffi-env-e2e` should run to install it.

Reply:
plan failed

The exception was caused by 1 earlier exceptions

Cause number 1:

    prepare step failed

    The exception was caused by 1 earlier exceptions

    Cause number 1:

        Command 'rpm -q --whatprovides podman || rpm-ostree install --apply-live --idempotent --allow-inactive  podman' returned 1.

        stdout (25 lines)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        no package provides podman
        Checking out tree 218334c...done
        Enabled rpm-md repositories: baseos appstream autosd extras-common
        Updating metadata for 'baseos'...done
        Updating metadata for 'appstream'...done
        Updating metadata for 'autosd'...done
        Updating metadata for 'extras-common'...done
        Importing rpm-md...done
        rpm-md repo 'baseos'; generated: 2024-09-02T14:56:42Z solvables: 4455
        rpm-md repo 'appstream'; generated: 2024-09-02T14:59:05Z solvables: 16992
        rpm-md repo 'autosd'; generated: 2024-09-03T02:37:15Z solvables: 6788
        rpm-md repo 'extras-common'; generated: 2024-08-06T11:03:43Z solvables: 93
        Resolving dependencies...done
        Will download: 20 packages (24.3?MB)
        Downloading from 'autosd'...done
        Importing packages...done
        Checking out packages...done
        Running pre scripts...done
        Running post scripts...done
        Running posttrans scripts...done
        Writing rpmdb...done
        Writing OSTree commit...done
        Staging deployment...done
        Computing /etc diff to preserve...done
        Updating /usr...done
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

        stderr (2 lines)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        notice: auto-inferring -y/--assumeyes when not run interactively; this will change in the future
        error: Changed directories are not supported yet
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~